.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+# Description
+A few sentences describing the overall goals of the pull request's commits.
+
+# GitHub Issues
+List the GitHub issues impacted by this PR:
+
+| GitHub Issue # |
+| -------------- |
+| |
+
+# Checklist:
+
+- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
+- [ ] I have verified that new and existing unit tests pass locally with my changes
+- [ ] I have not allowed coverage numbers to degenerate
+- [ ] I have maintained at least 90% code coverage
+- [ ] I have commented my code, particularly in hard-to-understand areas
```

Comment:
- [ ] I have commented on my code, particularly in hard-to-understand areas

Reply:
this check is already added in PR template, anything in particular to be addressed?

Reply:
@spriya-m : Please note the grammar check that has been corrected

Reply:
ok, addressed the comment.